guides/linting.md

```diff
@@ -0,0 +1,14 @@
+## Use Linting
+
+The linting tools are powerful aids to help identify common pitfalls and
+anti-patterns that can impede engineer and product success.  They also help
+streamline the review process, enabling engineers to provide quality feedback
+focused on solving the product and engineering problems at hand, as well as
+empowering the authoring engineer to maintain and improve code quality for
+their products.
+
+Suggested Tooling:
+
+* ESLint
+  * [eslint-plugin-ember-best-practices](https://github.com/ember-best-practices/eslint-plugin-ember-best-practices)
```

Comment:
I've received a lot of positive feedback on using `@ts-check` in visual studio code, w/ [@types/ember](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/ember). May be worth adding